astropy/stats/sigma_clipping.py

```diff
@@ -165,6 +165,13 @@ class SigmaClip:
         an ``axis`` keyword to return an array with axis dimension(s)
         removed.  The default is ``'std'``.
 
+    grow : float or `False`, optional
+        Radius within which to mask the neighbouring pixels of those that
+        fall outwith the clipping limits (only applied along `axis`, if
```

Comment:
```suggestion
        fall outwith the clipping limits (only applied along ``axis``, if
```

Reply:
Great, thanks. I changed that locally to avoid having 3 commits, before noticing the batch button, but I mentioned you in the commit message so maybe it will have the same effect.